spec/integration/kibana/knowledge_base_entry/manifest.spec.yml

```diff
@@ -0,0 +1,43 @@
+##
+## Describes the specification for a Kibana knowledge base entry's manifest.yml file
+##
+spec:
+  type: object
+  additionalProperties: false
+  properties:
+    name:
+      description: The name of the knowledge base entry
+      type: string
```

Comment:
Is `name` needed? Fleet could use the package and knowledge base directory names to build a unique one.

Reply:
I was planning on autogenerating an "id" based on the package name and KB directory, but the manifest's `name` is more of a meta data field for a user-readable "name", which will be displayed in the KB registry's UI.

Reply:
Ah ok, makes sense. Maybe then we could call it `label` or `title`? But as you prefer, name in that context sounds also good.

Reply:
I think you're right, `title` is probably more explicit and avoid mixing the concepts. Will update